.gitignore

```diff
@@ -15,3 +15,5 @@ api/clerk
 api/car
 test-results
 clients
+.vscode/*
```

Comment:
Let's actually just get rid of both so that everything in the dir is shared, anything secret that should be ignored should be moved up to user settings instead of workspace settings